src/output.rs

```diff
@@ -0,0 +1,66 @@
+use crate::card::Card;
+use crate::hand::Hand;
+use crate::hand_score::HandScore;
+use crate::hand_stats::HandStats;
+use poker_stats::aggregate_score::sample_aggregate_scores;
+use rand::rngs::ThreadRng;
+
+pub fn compute<const CARD_NUMBER: usize>(mut rng: ThreadRng) {
+    println!("");
+    let card_hand = Hand::<CARD_NUMBER>::draw(&mut rng);
+    println!("{card_hand}");
+    let hand_stats = HandStats::from(&card_hand);
+    println!("{hand_stats}");
+    let hand_score = HandScore::from(&hand_stats);
+    println!("{hand_score}");
+}
+
+pub fn print_sorted_deck(){
+    println!("Sorted Deck:");
+
+        for id in 0..52 {
+            let card = Card { id };
+
+            if card.suit().id == 3 {
+                println!("  {}", card);
+            } else {
+                print!("  {}  ", card);
+            }
+        }
+}
+
+pub fn draw_hand(hands_number: usize) {
```

Comment:
I'm not sure what to name this one... here is one idea. Trying to capture (1) what it does more clearly without clashing with the function that does the core implementation.
```suggestion
pub fn draw_and_display_hand_wrapper(hands_number: usize) {
```